src/components/Table/Table.tsx

```diff
@@ -29,7 +29,7 @@ export const TableRow: FC<PropsWithChildren<TableRowProps>> = ({
   return (
     <div
       className={clsx(
-        "relative grid grid-cols-[4fr,1fr,1fr,1fr,3.5rem] items-center gap-x-2 overflow-hidden rounded-md bg-black/60 p-3 text-xs backdrop-blur-md sm:text-sm md:gap-x-3 lg:px-6 lg:text-base",
+        "relative grid grid-cols-[4fr,1fr,1fr,1fr,3.5rem] items-center gap-x-2 overflow-hidden rounded-md bg-pink-900/80 p-3 text-xs backdrop-blur-md sm:text-sm md:gap-x-3 lg:px-6 lg:text-base",
```

Comment:
`bg-black/60` -> `bg-pink-900/80` is a massive improvement. Looks much closer to designs.